### PR TITLE
add store nodes and configure nim-waku

### PR DESCRIPTION
**Pull request by yakimant**

### CODE REVIEW

File: ansible/group_vars/store.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+# Custom SSH accounts for Shards fleet, should start from UID 8000.
```
Comment (jakubgs) [inline ansible/group_vars/store.yml:2]: Unnecessary comment and newline.

File: workspaces.tf [hunk 2]
```diff
@@ -12,6 +12,7 @@ locals {
       /* Default settings for all fleets/workspaces. */
 
       boot_hosts_count = 2
+      store_hosts_count = 2
```
Comment (jakubgs) [inline workspaces.tf:15]: Honestly, now that I think about it, do we really need two nodes per DC for the test fleet?

**Review by jakubgs — APPROVED** [review]

File: workspaces.tf (hunk 2 above)
Comment (yakimant) [reply workspaces.tf:15]: Maybe to test write operations to the DB?

Comment (jakubgs) [reply workspaces.tf:15]: Fair enough. But then we don't need two boot nodes do we?

Comment (yakimant) [reply workspaces.tf:15]: https://github.com/status-im/infra-shards/pull/11

**Pull request merged**